multiraft/multiraft.go

```diff
@@ -610,12 +612,14 @@ func (s *state) start() {
 						raft.DescribeMessage(req.Message, s.EntryFormatter))
 				}
 				s.handleMessage(req)
+				checkGroupID = req.GroupID
```

Comment:
I would get rid of `checkGroupID` and set `checkReadyGroupIDs[req.GroupID] = struct{}{}` here. This avoids any possible confusion about a `continue` or `return` between here and the end of the loop.